### PR TITLE
add logging ci to build collector release-5.6

### PR DESCRIPTION
**Pull request by jcantrill**
This PR:
* adds CI to build logging-fluentd for the release-5.6 branch
* adds CI to build vector for the release-5.6 branch

### CODE REVIEW

**Comment by jcantrill** [comment]
cc @xperimental

File: ci-operator/config/ViaQ/vector/ViaQ-vector-release-5.6.yaml [hunk 1]
```diff
@@ -0,0 +1,159 @@
+base_images:
+  builder:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "9"
```
Comment (xperimental) [inline ci-operator/config/ViaQ/vector/ViaQ-vector-release-5.6.yaml:5]: Should this be "8"?

**Pull request merged**